apps/frontend/src/libs/components/tab-item/tab-item.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+	type FieldValues,
+	type Path,
+	type UseFormRegister,
+} from "react-hook-form";
+
+import styles from "./styles.module.css";
+
+type TabContents = {
+	[key: string]: JSX.Element;
+};
+
+type Properties<T extends FieldValues> = {
+	label: string;
+	name: Path<T>;
+	register: UseFormRegister<T>;
+	selectedTab: string;
+	tab: string;
+	tabContents: TabContents;
```

Comment:
```suggestion
	tabsContent: TabContents;
```
We name arrays in plural